package.json

```diff
@@ -18,16 +18,19 @@
     "codecov": "^1.0.1",
     "eslint": "^3.5.0",
     "hapi": "^15.1.0",
+    "goodparts": "^1.0.3",
     "istanbul": "^0.4.5",
     "pre-commit": "^1.1.3",
     "tape": "^4.6.0"
   },
   "scripts": {
     "test": "node_modules/.bin/tape ./test/*.test.js",
-    "lint": "node_modules/.bin/eslint .",
-    "lint:fix": "node_modules/.bin/eslint . --fix",
+    "lint": "node_modules/.bin/goodparts .",
+    "lint:init": "ln -sf node_modules/goodparts/.eslintrc.js ./.eslintrc.js",
```

Comment:
See how lin and lint fix are handy scripts to add but feel like lint:init should be done on the postinstall of the goodparts module. Strange that those goodparts goodtime guys would expect you to add this script to your package.json before installing to get the linting kicked off..